### ISSUE TITLE
Regarding creation of texture objects

### ISSUE THREAD

**Issue by saketh2002** [ScheiklP/sofa_env#29]
Hi,
I followed the how to documents to create obj and msh files, using visual gallbladder of grasp lift touch environment, then I tried to replace the cloth in tissue retraction task with the obj and msh I created. I know it is an incomplete env, but I wanted to check if the texture actually works with other stl files. But I have been getting the following output. 
![Screenshot from 2024-06-07 07-17-27](https://github.com/ScheiklP/sofa_env/assets/65168029/623bb8b0-c8a1-4de6-93ba-5f7024513c18)

when I tried with vtk file instead of msh, I waited couple of hours for the output, but none appeared.  Could the issue might be with the msh conversion I did ? I got no error in gmsh or blender when creating files.

**Comment by ScheiklP** [comment]
Hi @saketh2002 ,
I answered the question in a previous issue already, right?
https://github.com/ScheiklP/sofa_env/issues/27

Cheers,
Paul

**Comment by saketh2002** [comment]
Hi @ScheiklP , I was able to get output for tissue retraction environment. But when changed the obj and msg files, essentially replacing the cloth with another object, I wasnt even able to get the tissue-comp not found in examples error.  Note that the error you are seeing in the screenshot, I didn't press a ctrl c , it had automatically aborted, unlike in tissue retraction with cloth setup, where I was able to get output after waiting for a while.
